nats-pure.gemspec

```diff
@@ -25,11 +27,33 @@ Gem::Specification.new do |s|
   s.authors = ['Waldemar Quevedo']
   s.email = ['wally@synadia.com']
 
+  s.metadata = {
+    "bug_tracker_uri" => "https://github.com/nats-io/nats-pure.rb/issues",
+    "changelog_uri" => "https://github.com/nats-io/nats-pure.rb/blob/master/CHANGELOG.md",
```

Comment:
think it would be `main` branch here, there is no `master` branch

Reply:
yeah, good catch; fixed